contracts/src/systems/combat/contracts.cairo

```diff
@@ -1366,6 +1366,11 @@ mod combat_systems {
             owner_armies_quantity.count -= 1;
             set!(world, (owner_armies_quantity));
 
+            let protectee: Protectee = get!(world, army.entity_id, Protectee);
```

Comment:
@credence0x is this check necessary, ie. are we ever deleting a defensive army somewhere else?

Reply:
no. but it should be there. it's fine